include/podio/ObjectID.h

```diff
@@ -5,6 +5,10 @@
 #include <iomanip>
 #include <ostream>
 
+#if defined(PODIO_JSON_OUTPUT) && !defined(__CLING__)
```

Comment:
we have to watch out for this combination of flags. If we move it more often, we should replace it with a single combined one.